handler/src/test/java/io/netty/handler/ssl/SSLEngineTest.java

```diff
@@ -4031,4 +4031,34 @@ protected SslContext wrapContext(SslContext context) {
     protected String[] protocols() {
         return new String[] { protocolCipherCombo.protocol };
     }
+
+    @Test
+    public void testRSASSAPSS() throws Exception {
```

Comment:
Should this be moved to `OpenSslEngineTest` or is it supported by the JDK as well ?

Reply:
As far as I know. This is now supported by oracle, zulu and openJDK.

Reply:
I can still move it to OpenSslEngineTest, if that makes more sense.

Reply:
I am just asking as the test fails on java8 and 11 it seems. Do you know in which version support was added ?

Reply:
oh. okay in my system I have Zulu installed but these fixes are done very recently in the past 6 months. 
https://bugs.java.com/bugdatabase/view_bug.do?bug_id=JDK-8146293

Reply:
So you say this test pass on your system ?

Reply:
Yes, I only saw one other test failing though.

Reply:
But I guess  For the sake of everyone's convenience it would be better to move it opensslEngineTest. what do you say?

Reply:
sure. I will make the change.

Reply:
moved the testcase to OpensslEngineTest.